docs/server/source/appendices/docker-on-mac.md

```diff
@@ -35,7 +35,7 @@ Install Docker for Mac.
 Pull the bigchaindb and other required docker images from docker hub.
 
 ```text
-docker pull bigchaindb/bigchaindb:master
+docker pull bigchaindb/bigchaindb:latest
```

Comment:
I was assuming "latest" is latest release?

Reply:
`latest` is the latest release i.e. built from 1.1 branch. Master is built from master codebase. I still am a bit unsure about the question :)

Reply:
I guess you could write:

```text
docker pull bigchaindb/bigchaindb:TAG
```

and then explain that TAG is one of the tag options for the bigchaindb/bigchaindb image listed on Docker Hub at https://hub.docker.com/r/bigchaindb/bigchaindb/tags/ . The image with the `master` tag was built using the current code of the BigchainDB master branch and `latest` is the same as the highest-numbered release.

Reply:
> latest is the latest release i.e. built from 1.1 branch. Master is built from master codebase. I still am a bit unsure about the question :)

That's all I wanted to know. Thx

> I guess you could write:

As with pip, we want users to download the latest release and not master.
I'll keep it as `latest`, OK?

Reply:
I guess there's a different docs page about using Docker for dev/test, so here it's probably fine if you tell them to use the `latest` tag

Reply:
👍

Reply:
OK. So can you give your approval then to have this merged?